src/cogs/cpp.py

```diff
@@ -52,28 +52,22 @@ async def cppref(self, ctx, *, query: str):
         description = []
         q = query.replace('std::', '')
 
-        if os.path.isdir(f"src/cppref/cpp/{q}"):
-            description.append(
-                f"[`std::{q}`](http://en.cppreference.com/w/cpp/{q})")
-
         for _, result in enumerate(results):
             result = result.replace("src/cppref/", "")
             check_name = result.replace(
                 "http://en.cppreference.com/w/", "")
             check_name = check_name.replace("\\", "/")
-            # print(check_name)
 
-            # print(check_name)
             f_name = check_name.replace("/", "::")
             f_name = f_name.replace(".html", "")
 
             if check_name.startswith(("language", "concept")) and not check_name.startswith("concepts"):
                 special_pages.append(
-                    f'[`{f_name}`]({result})')
+                    f'[`{f_name.replace("cpp::", "")}`]({result})')
```

Comment:
it's not just cpp::, there's a lot more. 
![image](https://user-images.githubusercontent.com/49172424/105059191-4dee4100-5a77-11eb-87b7-e4f3f253a2f1.png)

Reply:
So if it starts with std::{directory} -> replace

Reply:
What we could do is scan the directory for other directories, and add it to a list

Reply:
I hadn't thought of that. Upon further investigation, I have found that there is a `header` folder that contains the names of all the headers. So if the name is a file in that folder, I can remove it from the list. But then I run into the problem of stuff like `std::string`. There is a header `string` and a class `string`. I will have to think about a solution to this. If you have any ideas, feel free to share!